composer.json

```diff
@@ -35,7 +35,12 @@
         "codeception/specify": "~0.3",
         "squizlabs/php_codesniffer": "~2.0",
         "vlucas/phpdotenv": "^3.0",
-        "symfony/process": ">=2.7 <5.0"
+        "symfony/process": ">=2.7 <5.0",
+        "codeception/module-filesystem": "^1.0",
+        "codeception/module-cli": "^1.0",
+        "codeception/module-asserts": "^1.1",
+        "codeception/module-phpbrowser": "^1.0",
+        "codeception/util-universalframework": "@dev"
```

Comment:
Please remove `composer.json` changes.

We have a circular dependency problem here.
In order to be able to test branches of those modules we must either remove module dependencies from this repo or add very loose dependencies as I already did in those modules:
https://github.com/Codeception/module-asserts/commit/7f0e038316d84cefc4098ea570225bf9debc0304.

Reply:
I raised pull request for that: #5772

Reply:
Done!